insalan/tournament/views.py

```diff
@@ -189,6 +188,67 @@ def get(self, request, primary_key: int):
         return Response(tourney_serialized, status=status.HTTP_200_OK)
 
 
+class TournamentMe(APIView):
+    """ 
+    Details on tournament of a logged user
+    This endpoint does many requests to the database and should be used wisely
+    """
+    authentication_classes =  [SessionAuthentication]
+    permission_classes = [permissions.IsAuthenticated & ReadOnly]
+
+    def get(self, request):
+        user: User = request.user
+
+        if user is None:
+            raise PermissionDenied()
+
+        # retrieve inscription as Player
+        player_inscriptions = Player.objects.filter(user=user)
+        # serialize it
+        player_inscriptions = serializers.PlayerSerializer(
```

Comment:
The English word for "inscription" is "registration". Please change it and afterwards to keep a consistent and understandable code base.